doc/Yang Model Guidelines.md

```diff
@@ -140,27 +143,18 @@ container VLAN_MEMBER {
 }
 ```
 
-### 5.  It is best to categorized Yang Modules based on a networking components. For Example, it is good to have separate modules for VLAN,  ACL, PORT and IP-ADDRESSES etc.
+### 5.  It is best to categorized YANG Modules based on a networking components. For Example, it is good to have separate modules for VLAN,  ACL, PORT and IP-ADDRESSES etc.
 ```
 sonic-acl.yang
 sonic-interface.yang
 sonic-port.yang
 sonic-vlan.yang
 ```
 
-### 6. When it comes about the choice between objects, it is good to stick to standard models. 
-For example: if open-config defines acl-rules as container and have an acl-rules list inside it, then we will stick to the same object types, if a container and list is must in Sonic Yang model.
-Examples:
-```
-container acl-entries {
-    description "Access list entries container";
-    list acl-entry {
-        key "sequence-id";
-```
-
 
 
-### 7.  All must, when, pattern and enumeration constraints will come from .h files (or code). If code has the possibility to have unknown behavior with some config, then we will put a constraint in yang models objects. (This will always be Ongoing effort).
+### 6.  All must, when, pattern and enumeration constraints can be derived from .h files (or code). If code has the possibility to have unknown behavior with some config, then we should put a constraint in YANG models objects. Also, developr can put any additional constraint to stop invalid configuration.
```

Comment:
I believe we need to explain "unknown behavior with some config" a bit here, with an example. This will help developer to think on appropriate constraint in this context.

"Developer"...

Reply:
We could drop this line "_If code has the possibility to have unknown behavior with some config, then we should put a constraint in YANG models objects._ ", as this implies a ***missed*** validation in code.  This is part of reading code to understand constraints, some explicitly present and some implicit (like unknown behavior)